storage/local/storage.go

```diff
@@ -360,11 +371,28 @@ func (s *memorySeriesStorage) GetMetricForFingerprint(fp clientmodel.Fingerprint
 
 // AppendSamples implements Storage.
 func (s *memorySeriesStorage) AppendSamples(samples clientmodel.Samples) {
+	if len(samples) == 0 {
```

Comment:
Ok, doesn't matter much, but since I expect 0-length slices to (almost?) never happen, it seems like just doing useless extra work. Let me dig into why we had that check in the first place. Removing (unneeded) code is always better than keeping it...